hybridse/src/vm/runner.cc

```diff
@@ -1106,9 +1106,10 @@ std::shared_ptr<DataHandlerList> Runner::BatchRequestRun(RunnerContext& ctx) {
 
     for (size_t idx = 0; idx < ctx.GetRequestSize(); idx++) {
         inputs.clear();
-        for (size_t producer_idx = 0; producer_idx < producers_.size();
-             producer_idx++) {
-            inputs.push_back(batch_inputs[producer_idx]->Get(idx));
+        for (size_t producer_idx = 0; producer_idx < producers_.size(); producer_idx++) {
+            if (batch_inputs[producer_idx] != nullptr) {
+                inputs.push_back(batch_inputs[producer_idx]->Get(idx));
+            }
```

Comment:
input array size changed, not sure if problem arrise. or Construct a empty element instead of null ?